tabular/src/autogluon/tabular/models/tabular_nn/torch/tabular_nn_torch.py

```diff
@@ -37,6 +39,10 @@
 class TabularNeuralNetTorchModel(AbstractNeuralNetworkModel):
     """
     PyTorch neural network models for classification/regression with tabular data.
+
```

Comment:
qq: in reference to -> Switch default ES to use adaptive early stopping
this switch is just being made to NN_Torch correct ?

Reply:
Yes only NN_TORCH